lib/supermarket/curry_mass_subscribe.rb

```diff
@@ -0,0 +1,32 @@
+module Supermarket
+  require 'octokit'
+
+  class CurryMassSubscribe
+    def subscribe_org_repos(org)
+      public_repos(org).each do |repo|
+        if Curry::Repository.where(owner: org, name: repo[:name]).present?
+          puts "#{repo[:name]} is already subscribed.  Skipping..."
+        else
+          repository = Curry::Repository.new(owner: org, name: repo[:name])
+          subscriber = Curry::RepositorySubscriber.new(repository)
+
+          if subscriber.subscribe('https://supermarket.chef.io/curry/pull_request_updates')
+            Curry::RepositorySubscriptionWorker.perform_async(repository.id)
+          else
+            puts "Unable to subscribe #{repository.name}"
+          end
+        end
+      end
+    end
+
+    private
+
+    def client
+      Octokit::Client.new(access_token: ENV['GITHUB_ACCESS_TOKEN'])
+    end
+
+    def public_repos(org)
+      client.org_repos(org, private: true)
```

Comment:
There might be something amiss with this line, even though the script seemed to run fine.  I feel like it should have said client.org_repos(org, private: false) to return public repos.  However, when I ran this line individually, it definitely picked up public repos (and returned the same number of repos regardless of whether we specified private being true or false), so there might be something incorrect with how we are calling the org repos and trying to filter them so that we only get the public ones.  I'm not merging this script into the code base, so I'm going to go ahead and close this pull request and leave the script as it is, but I will leave this note as a caution to anyone who might come across this pull request in the future.